docs/source/advanced/experimental-websockets.mdx

```diff
@@ -26,12 +26,94 @@ The `com.apollographql.apollo.network.websocket` implementation provides the fol
 
 ## Status
 
-While they are `@ApolloExperimental`, we believe the new `.websocket` APIS to be more robust than the non-experimental `.ws` ones. They are safe to use in non-lib use cases.
+`.websocket` APIS to be more robust than the non-experimental `.ws` ones, especially in scenarios involving retries/connection errors. They are safe to use in non-lib use cases.
```

Comment:
```suggestion
We believe the `.websocket` APIs to be more robust than the non-experimental `.ws` ones, especially in scenarios involving retries/connection errors. They are safe to use in non-lib use cases.
```

Reply:
I wanted to remove the "belief" from that sentence. We have evidence now! Updated in https://github.com/apollographql/apollo-kotlin/pull/6275/commits/d08c5f7ffae64ac935a180a1cc5954333bd22a9a